README.md

```diff
@@ -1,94 +1,122 @@
-[![Apache License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-# Marketo Source ([docs](https://fivetran-dbt-marketo.netlify.app/#!/overview))
+<p align="center">
+    <a alt="License"
+        href="https://github.com/fivetran/dbt_marketo_source/blob/main/LICENSE">
+        <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" /></a>
+    <a alt="dbt-core">
+        <img src="https://img.shields.io/badge/dbt_Core™_version->=1.0.0_,<2.0.0-orange.svg" /></a>
+    <a alt="Maintained?">
+        <img src="https://img.shields.io/badge/Maintained%3F-yes-green.svg" /></a>
+    <a alt="PRs">
+        <img src="https://img.shields.io/badge/Contributions-welcome-blueviolet" /></a>
+</p>
 
-This package models Marketo data from [Fivetran's connector](https://fivetran.com/docs/applications/marketo). It uses data in the format described by [this ERD](https://fivetran.com/docs/applications/marketo#schema).
+# Marketo Source dbt Package ([docs](https://fivetran.github.io/dbt_market_source/))
+# 📣 What does this dbt package do?
 
-This package enriches your Fivetran data by doing the following:
+- Produces staging tables that leverage Marketo data from [Fivetran's connector](https://fivetran.com/docs/applications/marketo) in the format described by [this ERD](https://fivetran.com/docs/applications/marketo#schema).
+  - Adds descriptions to tables and columns that are synced using Fivetran
+  - Models staging tables, which will be used in our transform package
+  - Adds column-level testing where applicable. For example, all primary keys are tested for uniqueness and non-null values.
+- Generates a comprehensive data dictionary of your source and modeled Marketo data through the [dbt docs site](https://fivetran.github.io/dbt_marketo_source/).
+- These tables are designed to work simultaneously with our [Marketo transformation package](https://https://github.com/fivetran/dbt_marketo/).
```

Comment:
It looks like you have two `https` instances in this link.

Reply:
Thanks done!